src/main/io/osd.c

```diff
@@ -251,34 +251,41 @@ static void osdLeftAlignString(char *buff)
  */
 static void osdFormatDistanceSymbol(char *buff, int32_t dist, uint8_t decimals)
 {
+    uint8_t digits = 3U;    // Total number of digits (including decimal point)
+    uint8_t sym_index = 3U; // Position (index) at buffer of units symbol
+    if (isBfCompatibleVideoSystem(osdConfig())) {
```

Comment:
It may be worth adding the pre-processor check to this if block, and the other similar blocks. Eg.
```
#ifndef DISABLE_MSP_BF_COMPAT
    if (isBfCompatibleVideoSystem(osdConfig())) {
        // Add one digit so up no switch to scaled decimal occurs above 99
        digits = 4U;
        sym_index = 4U;
    }
#endif
```
There is no need for the check or changing the values if BF43COMPAT is not supported.

Reply:
It may not make much of a difference on the final binary. if Bf compat is disabled, IsBfCompatible is replaced by a constant false and the block is likely to be optimized out by the compiler.

Reply:
In that case, I guess the pre-processor checks won't do any harm, so I'm inclined to keep them in the proposed mods. I'm waiting for the weather to improve so I can put in some test flights and verify all the values work as intended before removing the draft status.